robot/description/urdf/casters.urdf.xacro

```diff
@@ -10,25 +10,24 @@
         <visual>
             <origin rpy="0 0 0" xyz="0 0 0"/>
             <geometry>
-                <mesh filename="package://description/urdf/meshes/front_caster.dae"/>
+                <sphere radius="${caster_link_radius}"/>
             </geometry>
         </visual>
 
         <collision>
             <origin rpy="0 0 0" xyz="0 0 0"/>
             <geometry>
-                <mesh filename="package://description/urdf/meshes/front_caster.dae"/>
+                <sphere radius="${caster_link_radius}"/>
             </geometry>
         </collision>
 
         <inertial>
             <mass value="${caster_link_mass}"/>
-            <xacro:caster_link_inertia/>
+            <xacro:sphere_inertia mass="${caster_link_mass}" radius="${caster_link_radius}"/>
         </inertial>
     </link>
 
-    <joint name="${tf_prefix}/front_caster_link_joint" type="continuous">
-        <axis xyz="0 0 1"/>
+    <joint name="${tf_prefix}/front_caster_link_joint" type="fixed">
```

Comment:
So a question is why doesn't a `floating` joint work and we have to use a `ball` joint? 

Nonetheless, I believe that the type of this specific joint should replace `fixed` with `floating` because it is more representative of the type of joint we are trying to use.

Reply:
See: https://answers.ros.org/question/74626/i-get-a-warnning-when-trying-to-use-a-floating-joint/

When using floating joint, I got the same warning. It seems to be no longer supported.